.ci/jobs/e2e-stack-versions.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Isn't this the exact same file as `gke-e2e-versions.yaml`?
Would it be worth it having a single job that we reuse for both use cases?